src/utils/blockchain.ts

```diff
@@ -238,6 +255,10 @@ export const getExplorerFeeEstimations = async (asset: string) => {
 
         const expectedBlocksToConfirm = "3";
 
+        if (typeof feeEstimations[expectedBlocksToConfirm] !== "number") {
+            handleUnexpectedResponse(feeEstimations);
```

Comment:
Throwing inside of the function that fetches the estimation means you don't have to remember and repeat this logic everywhere you use it